src/fswAlgorithms/imageProcessing/centerOfBrightness/_UnitTest/test_centerOfBrightness.py

```diff
@@ -0,0 +1,186 @@
+
+ # ISC License
+ #
+ # Copyright (c) 2016, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
+ #
+ # Permission to use, copy, modify, and/or distribute this software for any
+ # purpose with or without fee is hereby granted, provided that the above
+ # copyright notice and this permission notice appear in all copies.
+ #
+ # THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ # WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ # MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ # ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ # WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ # ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ # OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+
+import pytest
+import os, inspect, glob
+import numpy as np
+
+filename = inspect.getframeinfo(inspect.currentframe()).filename
+path = os.path.dirname(os.path.abspath(filename))
+bskName = 'Basilisk'
+splitPath = path.split(bskName)
+
+# Import all of the modules that we are going to be called in this simulation
+importErr = False
+reasonErr = ""
+try:
+    from PIL import Image, ImageDraw
+except ImportError:
+    importErr = True
+    reasonErr = "python Pillow package not installed---can't test CenterOfBrightness module"
+
+# Import all of the modules that we are going to be called in this simulation
+from Basilisk.utilities import SimulationBaseClass, unitTestSupport
+from Basilisk.utilities import macros
+from Basilisk.architecture import messaging
+
+try:
+    from Basilisk.fswAlgorithms import centerOfBrightness
+except ImportError:
+    importErr = True
+    reasonErr = "Center Of Brightness not built---check OpenCV option"
+
+# Uncomment this line is this test is to be skipped in the global unit test run, adjust message as needed.
+# @pytest.mark.skipif(conditionstring)
+# Uncomment this line if this test has an expected failure, adjust message as needed.
+# @pytest.mark.xfail(conditionstring)
+# Provide a unique test method name, starting with 'test_'.
+
+@pytest.mark.skipif(importErr, reason= reasonErr)
+@pytest.mark.parametrize("image, blur,  saveTest, valid, saveImage", [
```

Comment:
:large_blue_circle: I was curious, if we don't vary `blur`, 'valid', or 'saveimage`, why do we expose them as parameters instead of assigning them directly?

Reply:
We should, I'll improve the test